cmd/skopeo/inspect.go

```diff
@@ -115,14 +118,21 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 	}
 
 	if opts.config && opts.raw {
-		var configBlob []byte
+		var (
+			configBlob []byte
+			err        error
+		)
 		if err := retry.RetryIfNecessary(ctx, func() error {
 			configBlob, err = img.ConfigBlob(ctx)
 			return err
 		}, opts.retryOpts); err != nil {
 			return errors.Wrapf(err, "Error reading configuration blob")
 		}
-		_, err = stdout.Write(configBlob)
+		if len(opts.format) == 0 {
```

Comment:
If this affects `--config --raw`, it should affect a non-config `--raw` as well.

Reply:
As far as I can see the --raw option seems to have no effect currently if the --config flag is not specified.  Is it supposed to?

With the current code these commands have the same output.
```
/bin/skopeo  inspect --raw=true --raw docker://docker.io/golang 
/bin/skopeo  inspect --raw=false --raw docker://docker.io/golang 
```

Reply:
> With the current code these commands have the same output.

That’s not what I see:
```console
% bin/skopeo --override-os linux inspect --raw=true docker://docker.io/golang  
{"manifests":[{"digest":"sha256:097…

% bin/skopeo --override-os linux inspect --raw=false docker://docker.io/golang 
{
    "Name": "docker.io/library/golang",
…
```

Reply:
Yes you are right.  My mistake.